2.7.pylintrc

```diff
@@ -0,0 +1,25 @@
+
+[MESSAGES CONTROL]
+
+disable=bad-option-value,misplaced-comparison-constant,no-self-use,consider-iterating-dictionary,
```

Comment:
can we add a short description of each suppresion?

Reply:
@kevinclark19a - my main goal is to make it easy to fix the code... i'm thinking we would come to this file, read the description of the issues, pick up one, highlight the code or symbol, do a global search on the code using the highlighted text, and then go fix each instance